spec/tracker/tracker_spec.rb

```diff
@@ -12,6 +12,11 @@ class BodyHandler < DummyHandler
   self.position = :body
 end
 
+class BodyOpeningHandler < DummyHandler
+  self.position = :body
+  self.exact_position = :opening
```

Comment:
Yeah. I wasn't sure about the semantics here either. But I wanted to just get this PR out to get feedback about this.
:container/:position actually sound better. Thanks :)